common/config/subspaces/build-tests-subspace/common-versions.json

```diff
@@ -49,6 +49,21 @@
    */
   // "implicitlyPreferredVersions": false,
 
+  /**
+   * If you would like the version specifiers for your dependencies to be consistent, then
+   * uncomment this line. This is effectively similar to running "rush check" before any
+   * of the following commands:
+   *
+   *   rush install, rush update, rush link, rush version, rush publish
+   *
+   * In some cases you may want this turned on, but need to allow certain packages to use a different
+   * version. In those cases, you will need to add an entry to the "allowedAlternativeVersions"
+   * section of the common-versions.json.
+   *
+   * In the case that subspaces is enabled, this setting will take effect at a subspace level.
+   */
+  "ensureConsistentVersions": false,
```

Comment:
Why?

Reply:
That's just what was in the config file prior to this upgrade.  I can try enabling it.

Reply:
Good catch - I enabled it and `rush update` succeeds.  

Also I noticed the online schema flags it as an error; I'll make a PR to update the schemas repo.